Source/WTF/wtf/WeakHashSet.h

```diff
@@ -156,6 +156,13 @@ class WeakHashSet final {
         return m_set.size();
     }
 
+    void swap(WeakHashSet& other)
```

Comment:
I bet we can make the code use move semantics instead of `swap`. Using `swap` was a sort of workaround back when we didn't have move semantics in C++.

While there's no harm in supporting `swap` I think it's likely not the best idiom, just pre-move code.

For example putting in a new empty set while pulling the old value out is clearest with `std::exchange`. But in the old days we would have used `swap`.

Reply:
Ok, switched to `std::exchange()` instead.